Makefile

```diff
@@ -125,8 +126,9 @@ lint: $(MYGOBIN)/golangci-lint $(MYGOBIN)/goimports $(builtinplugins)
 	./hack/for-each-module.sh "make lint"
 
 .PHONY: test-unit-all
-test-unit-all: $(builtinplugins)
-	./hack/for-each-module.sh "make test"
```

Comment:
It would be nice to use this definition, but it doesn't work right now. The plugin script is doing some setup / exlusions.